README.md

```diff
@@ -112,20 +115,30 @@ Creates a Peer ID from a buffer representing the key's multihash.
 ### `createFromB58String(str)`
 Creates a Peer ID from a Base58 string representing the key's multihash.
 
-### `createFromPubKey(pubKey)`
+### `createFromPubKey(pubKey, cb)`
+
+- `publicKey: Buffer`
+- `cb: Function`
+
+Creates a Peer ID from a buffer containing a public key and
+calls back `cb` with `err, id`.
 
-Creates a Peer ID from a buffer containing a public key.
+### `createFromPrivKey(privKey, cb)`
 
-### `createFromPrivKey(privKey)`
+- `privKey: Buffer`
+- `cb: Function`
 
-Creates a Peer ID from a buffer containing a private key.
+Creates a Peer ID from a buffer containing a private key and
+calls back `cb` with `err, id`.
 
-### `createFromJSON(obj)`
+### `createFromJSON(obj, cb)`
```

Comment:
Because it uses `createFromPubKey` and `createFromPrivKey`, but I just realised they don't have to be async either, not even when using webcrypto in the future, so I can revert these :) Thank you for questioning me